src/tests/ftest/container/list_containers.py

```diff
@@ -0,0 +1,110 @@
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+'''
+from apricot import TestWithServers
+from dmg_utils import get_pool_uuid_service_replicas_from_stdout
+from daos_utils import DaosCommand
+
+
+class ListContainerTest(TestWithServers):
+    """
+    Test Class Description:
+        Tests daos pool list-cont. Test the following cases.
+        1. Create 1 container, list, and verify the listed container UUID
+            matches the UUID returned when created.
+        2. Create 1 more container and verify list; 2 total containers
+        3. Create 98 more containers and verify list; 100 total containers
+        4. Create 2 additional pools and create 10 containers in each pool.
+            Verify the container UUIDs in these 2 new pools.
+
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize a ListContainerTest object."""
+        super(ListContainerTest, self).__init__(*args, **kwargs)
+        self.daos_cmd = None
+
+    def create_list(self, count, pool_uuid, sr, expected_uuids):
+        """Create container and call daos pool list-cont to list and verify.
+
+        Args:
+            count (Integer): Number of containers to create.
+            pool_uuid (String): Pool UUID to create containers in.
+            sr (String): Service replicas of the pool to create containers.
+            expected_uuids (List of string): List that contains container UUID.
+                It should contain all the container UUIDs for the given pool.
+        """
+        # Create containers and store the container UUIDs into expected_uuids.
+        for _ in range(count):
+            create_stdout = self.daos_cmd.container_create(
+                pool=pool_uuid, svc=sr).stdout
+            expected_uuids.append(create_stdout.split()[3])
+        expected_uuids.sort()
+        # daos pool list-cont returns the date, host name, and container UUID
+        # as below:
+        # 03/31-21:32:24.53 wolf-3 2f69b198-8478-472e-b6c8-02a451f4de1b
+        kwargs = {"pool": pool_uuid, "svc": sr}
```

Comment:
Personally I think using a regex is safer than grabbing a space-separated index, just in case something is wonky with the command output:
```suggestion
        kwargs = {"pool": pool_uuid, "svc": sr}
        for _ in range(count):
            expected_uuids.append(
                self.daos_cmd.get_output("container_create", **kwargs))
        expected_uuids.sort()
        # daos pool list-cont returns the date, host name, and container UUID
        # as below:
        # 03/31-21:32:24.53 wolf-3 2f69b198-8478-472e-b6c8-02a451f4de1b
```

Reply:
Done. Thanks for catching.